src/app/profile/settings/feature-opt-in/feature-opt-in.component.ts

```diff
@@ -145,6 +152,10 @@ export class FeatureOptInComponent implements OnInit, OnDestroy {
     }));
   }
 
+  onChange(event) {
```

Comment:
Can you add type annotations here?

Reply:
what annotations are you talking about? #puzzled but open to suggestions if more precise :)

Reply:
Like `onChange(event: Event): void {`

Reply:
ah oki, this event is a custom event coming from toggle component. let me add more type info tho.